src/morphs.jl

```diff
@@ -148,8 +148,8 @@ function save_morph_polygons!(action::Action, from_func::Function, to_func::Func
         end
     end
     new_from_poly = copy(from_poly)
-    for i in 1:length(from_poly)
-        new_from_poly[i] = from_poly[(i+smallest_i-1) % length(from_poly) + 1]
+    for i = 1:length(from_poly)
+        new_from_poly[i] = from_poly[(i+smallest_i-1)%length(from_poly)+1]
```

Comment:
I also prefer `in` as `=` should be for an assignment in my opinion

Reply:
I agree! There is a way to fix this using the kwarg, `always_for_in` set to true when formatting.